### DISCUSSION TITLE
Create table statement

### DISCUSSION THREAD

**Discussion by BobInCT** [dbeaver/dbeaver#10530] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version
- DBeaver version
Latest community
#### Connection specification:
- Database name and version
Poistgres
#### Describe your question:
Is there a way to generate a create table statement for a table already in the database?

**Comment by uslss** [comment, marked as answer]
Hi @BobInCT ,
you can do it using:
- context menu for the table in the database navigator
![image](https://user-images.githubusercontent.com/31996417/100572323-394ac380-32e6-11eb-8886-72801e837abc.png)
- table properties - DDL tab
![image](https://user-images.githubusercontent.com/31996417/100572362-5384a180-32e6-11eb-8ce8-aa71a9270664.png)

**Comment by BobInCT** [comment]
Ahhh, somehow missed that, thanks.
